docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
-## Quickstart
 
-Hi! Glad to see you made it to the Quickstart guide! In this guide you are introduced to how CARTOframes can be used by data scientists in spatial analysis workflows. Using bike share data, this guide walks through some common steps a data scientist takes to answer the following question: are the company's bike share stations placed in optimal locations?
+# Quickstart
 
-Before you get started, we encourage you to have your environment ready so you can get a feel for the library by using it. If you don’t have your environment set-up yet, check out this guide first. You will need:
+## Introduction
+Hi! Glad to see you made it to the Quickstart guide! In this guide you are introduced to how CARTOframes can be used by data scientists in spatial analysis workflows. Using fake Starbucks revenue data, this guide walks through some common steps a data scientist takes to answer the following question: which stores are performing better than others?
 
-- A python notebook environment
-- The CARTOframes library installed
+Before you get started, we encourage you to have CARTOframes installed so you can get a feel for the library by using it:
+
+```
+pip install --pre cartoframes
+```
+
+If you don’t have your python environment set-up yet, check out this guide first.
```

Comment:
add link to guide

Reply:
I will change it with "If you want to know more ways to install it, check out the [installation guide]() first."